### ISSUE TITLE
Operators <<?~ and <<?= don't exist

### ISSUE THREAD

**Issue by duog** [ekmett/lens#722]
Hi,
I reached for these operators recently and discovered they didn't exist. Since <?~ and <?= do exist, is there a reason these are missing?

I would expect them to have the following signatures (by symmetry):
```Haskell
(<<?~) :: LensLike ((,) a) s t a (Maybe b) -> b -> s -> (a, t)
(<<?=) :: MonadState s m => LensLike ((,) a) s s a (Maybe b) -> b -> m a
```

I would be happy to submit a pr if you agree these two operators would be good additions.

**Comment by Taneb** [comment]
Most of the `<<X~` and `<<X=` operators were written by me in the heady days of 2013. It's entirely possible I missed these, or the root operators were added later. Naturally, I'm in favour of adding these, with the philosophy that they're expected and at least one person finds them useful.

**Comment by phadej** [comment]
@duog please make a PR, could you add the operators mentioned in https://github.com/ekmett/lens/issues/719 at the same time? that would be great!

**Comment by duog** [comment]
Sure thing, will do.

**Comment by RyanGlScott** [comment]
Fixed in #725.
